debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+                        color: '#2b2b2b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    water_name: {
+                        weight: 1,
+                        fillColor: '#022c5b',
+                        color: '#022c5b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    transportation_name: {
+                        weight: 1,
+                        fillColor: '#bc6b38',
+                        color: '#bc6b38',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    place: {
+                        weight: 1,
+                        fillColor: '#f20e93',
+                        color: '#f20e93',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    housenumber: {
+                        weight: 1,
+                        fillColor: '#ef4c8b',
+                        color: '#ef4c8b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    poi: {
+                        weight: 1,
+                        fillColor: '#3bb50a',
+                        color: '#3bb50a',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    earth: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#c0c0c0',
+                        color: '#c0c0c0',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    }
+                };
+
+                // Monkey-patch some properties for nextzen layer names, because
+                // instead of "building" the data layer is called "buildings" and so on
+                vectorTileStyling.buildings  = vectorTileStyling.building;
+                vectorTileStyling.boundaries = vectorTileStyling.boundary;
+                vectorTileStyling.places     = vectorTileStyling.place;
+                vectorTileStyling.pois       = vectorTileStyling.poi;
+                vectorTileStyling.roads      = vectorTileStyling.road;
+
+            	// Nextzen tiles PBF layer setup
+
+                // Assumes layers = "all", and format = "mvt"
+                var nextzenTilesUrl = "https://tile.nextzen.org/tilezen/vector/v1/512/all/{z}/{x}/{y}.mvt?api_key={apikey}";
+
+                var nextzenVectorTileOptions = {
+                    rendererFactory: L.canvas.tile,
+                    attribution: '<a href="https://nextzen.com/">&copy; NextZen</a>, <a href="http://www.openstreetmap.org/copyright">&copy; OpenStreetMap</a> contributors',
+                    vectorTileLayerStyles: vectorTileStyling,
+                    apikey: 'gCZXZglvRQa6sB2z7JzL1w',
```

Comment:
Where did this key come from? Is it your personal key? Should we be including it in our repo?

Reply:
This key was copied from the Leaflet.VectorGrid sample code (https://leaflet.github.io/Leaflet.VectorGrid/demo-vectortiles.html). This source is acknowledged in an earlier comment.

Reply:
I don't think we should be using or promoting the use of an api key we don't own. Does nextzen have any free layers? If not, we might want to replace this with a different example.

Reply:
I'm leaning towards removing the Nextzen layer then - what do you think? The Leaflet protobuf examples (https://leaflet.github.io/Leaflet.VectorGrid/demo-vectortiles.html) all use a key of some kind except for Esri.

Reply:
Yeah, we should probably remove the Nextzen layer the then.

Reply:
I'll remove the Nextzen layer, and any layer styles that aren't used in the Esri layer.